tools/test-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This tool is used to setup the environment for running the tests. Its name
+# name and location is based on Zuul CI, which can automatically run it.
```

Comment:
ZuulCI is not something that we are running or using, so remove that part of this comment